specification/resources/resource-manager/Microsoft.Solutions/stable/2019-03-01/managedapplications.json

```diff
@@ -0,0 +1,1885 @@
+{
```

Comment:
In the future, please copy the last reviewed API version into the new version folder as the first commit. This enables an expedited review as we can diff the changes between commits.